homeassistant/components/epic_games_store/__init__.py

```diff
@@ -0,0 +1,50 @@
+"""The Epic Games Store integration."""
+from __future__ import annotations
+
+from epicstore_api import EpicGamesStoreAPI
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import CONF_LOCALE, DOMAIN
+from .coordinator import EGSUpdateCoordinator
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+def get_country_from_locale(locale: str) -> str:
+    """Get the country code from locale."""
+    excepts = {"ja": "JP", "ko": "KR", "zh-Hant": "CN"}
+    return (
+        excepts[locale]
+        if excepts.get(locale)
+        else (locale[3:] if ("-" in locale) else locale)
+    ).upper()
```

Comment:
Should this not be part of the `epicstore_api` library?

Reply:
This may, but don't know if it will be accepted as it won't be used anywhere else in the API code.

Reply:
Hummmm, I don't get the issue here, if you want your language to be German just select the `de` locale, the Game title & description will be German.

I created an auto locale because the country is used mainly for EGS statistics, and required by the EGS API, but otherwise useless.

Maybe, the currency of the price before it was free is affected, but indeed, in all Europe it's € (so it's not an issue in that case) and even for other countries, when you ask the data for a certain language it's because you are used to live there an use the native currency.

I don't think we need to change this, you can create multiple EGS sensors with different locales if you want the data in different languages & currencies ! 😉